charts/pega/templates/_helpers.tpl

```diff
@@ -239,7 +255,18 @@ true
 - name: {{ template "pegaVolumeCredentials" }}
   secret:
     # This name will be referred in the volume mounts kind.
-    secretName: {{ template "pegaCredentialsSecret" }}
+    secretName: {{ template "pegaCredentialsSecret" $ }}
     # Used to specify permissions on files within the volume.
     defaultMode: 420
 {{- end}}
+
+{{- define "deploymentName" }}{{ $deploymentNamePrefix := "pega" }}{{ if (.Values.global.deployment) }}{{ if (.Values.global.deployment.name) }}{{ $deploymentNamePrefix = .Values.global.deployment.name }}{{ end }}{{ end }}{{ $deploymentNamePrefix }}{{- end }}
+
+#As the pegasearch chart includes an externalURL value that is hardcoded to the name of the search service
```

Comment:
I'm not sure how this challenge observation helps...do you need to make a helpful, definitive statement or can you drop this comment? It's not clear what you are declaring other than a future problem if you need to rename the search service. Perhaps state the Pega recommendation and leave it that it can't be changed?